docs/docs/integrations/chat/fireworks.ipynb

```diff
@@ -147,7 +147,7 @@
    "source": [
     "# Tool Calling\n",
     "\n",
-    "Fireworks offers the [`FireFunction-v1` tool calling model](https://fireworks.ai/blog/firefunction-v1-gpt-4-level-function-calling). You can use it for structured output and function calling use cases:"
+    "Fireworks offers the [`FireFunction-v2` tool calling model](). You can use it for structured output and function calling use cases:"
```

Comment:
needs a url to go to

Reply:
ya, was waiting for link from Fireworks, but looks like its not coming in time. will just remove link for now.